packages/common/src/authorization/organizationMemberAbility.ts

```diff
@@ -114,6 +197,37 @@ const applyOrganizationMemberStaticAbilities: Record<
         can('create', 'DashboardComments', {
             organizationUuid: member.organizationUuid,
         });
+
+        // Dashboards
+        can('manage', 'Dashboard', {
+            organizationUuid: member.organizationUuid,
+            userSpaceAccess: {
+                $elemMatch: {
+                    userUuid: member.userUuid,
+                    role: {
+                        $in: [SpaceMemberRole.EDITOR, SpaceMemberRole.ADMIN],
```

Comment:
how does the granularity on space access relate to this?

<img width="202" alt="image" src="https://github.com/user-attachments/assets/d316e9f8-1654-436f-87fa-4ec6853d6989" />

Reply:
yes! 

can view = VIEWER
can edit = EDITOR
full access = ADMIN
Remove access = not in access array